app/src/main/java/com/github/epfl/meili/posts/PostListRecyclerAdapter.kt

```diff
@@ -8,38 +8,42 @@ import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
 import com.github.epfl.meili.R
 import com.github.epfl.meili.models.Post
+import com.github.epfl.meili.models.User
+import com.github.epfl.meili.util.ClickListener
 import com.github.epfl.meili.util.MeiliRecyclerAdapter
+import com.github.epfl.meili.util.MeiliWithUserRecyclerViewHolder
+import de.hdodenhof.circleimageview.CircleImageView
```

Comment:
These imports are not used, are you planning to use them later?

Reply:
Removed! Thanks!